distbench_threadpool.cc

```diff
@@ -13,10 +13,14 @@
 // limitations under the License.
 
 #include "distbench_threadpool.h"
+#include "glog/logging.h"
```

Comment:
not needed if we remove the logging statement below

Reply:
Done